src/components/Registration.tsx

```
@@ -1,10 +1,27 @@
+import { useMutation } from '@tanstack/react-query';
+import { ArrowLeft, User, Phone, Mail, MapPin, Camera, CheckCircle, Loader2 } from 'lucide-react';
 import { Button } from '@/components/ui/button';
 import { Card } from '@/components/ui/card';
 import { Input } from '@/components/ui/input';
 import { Label } from '@/components/ui/label';
 import { useToast } from '@/hooks/use-toast';
+import { useStore } from '@/store/useStore';
+
+const API_BASE_URL = import.meta.env.VITE_API_BASE_URL || '';
+
+const registerUser = async (data: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)